assets/javascripts/discourse/components/ai-persona-editor.gjs

```diff
@@ -433,20 +379,56 @@ export default class PersonaEditor extends Component {
           disabled={{this.editingModel.system}}
         />
       </div>
-      <div class="control-group">
-        <label>{{I18n.t "discourse_ai.ai_persona.max_context_posts"}}</label>
-        <Input
-          @type="number"
-          lang="en"
-          class="ai-persona-editor__max_context_posts"
-          @value={{this.editingModel.max_context_posts}}
+      {{#if this.editingModel.user}}
```

Comment:
All the checkbox changes here look good, definitely the way to go rather than toggles for this stuff :+1: